CODEOWNERS

```diff
@@ -4,90 +4,77 @@
 /applications/matter_weather_station/     @nrfconnect/ncs-matter
-/applications/nrf_desktop/                @MarekPieta
 /applications/nrf5340_audio/              @nrfconnect/ncs-audio
-/applications/serial_lte_modem/           @SeppoTakalo @MarkusLassila @rlubos @tomi-font
+/applications/nrf_desktop/                @MarekPieta
+/applications/serial_lte_modem/           @nrfconnect/ncs-co-networking @SeppoTakalo @MarkusLassila @tomi-font
 /applications/zigbee_weather_station/     @milewr
+/applications/**/*.rst                    @nrfconnect/ncs-doc-owners
+
 # Boards
-/boards/                                  @anangl
-/boards/nordic/thingy91*                  @anangl @nrfconnect/ncs-cia
+/boards/                                  @nrfconnect/ncs-co-boards
+/boards/nordic/thingy91*                  @nrfconnect/ncs-co-boards @nrfconnect/ncs-cia
+
 # All cmake related files
-/cmake/                                   @tejlmand
-/CMakeLists.txt                           @tejlmand
-# All Kconfig related files
-Kconfig*                                  @tejlmand
-# Sysbuild related files
-/sysbuild/                                @tejlmand @nordicjm
-/cmake/sysbuild/                          @tejlmand @nordicjm
+/cmake/                                   @nrfconnect/ncs-co-build-system
+
 # All doc related files
-/doc/_extensions/                         @gmarull
-/doc/_scripts/                            @carlescufi
-/doc/_static/                             @carlescufi
-/doc/_utils/                              @gmarull
-/doc/cache.yml                            @gmarull
-/doc/CMakeLists.txt                       @carlescufi
-/doc/**/conf.py                           @carlescufi
-/doc/kconfig/                             @gmarull
-/doc/nrf/                                 @carlescufi
+/doc/					  @nrfconnect/ncs-co-doc
+/doc/tags.yml                             @nrfconnect/ncs-co-doc @umapraseeda @b-gent
+/doc/**/*.rst				  @nrfconnect/ncs-doc-owners
+/doc/**/*.svg				  @nrfconnect/ncs-doc-owners
+/doc/**/*.png				  @nrfconnect/ncs-doc-owners
+/doc/nrf/protocols/thread/*.rst           @wiba-nordic
 /doc/nrf/device_guides/working_with_nrf/nrf54h/  @FrancescoSer
 /doc/nrf/device_guides/working_with_nrf/nrf54l/ @annwoj
-/doc/nrfx/                                @gmarull
-/doc/matter/                              @gmarull
-/doc/mcuboot/                             @carlescufi
-/doc/nrfxlib/                             @gmarull
-/doc/versions.json                        @carlescufi
-/doc/custom.properties                    @gmarull
-/doc/tags.yml                             @gmarull @umapraseeda @b-gent
-/doc/requirements.txt                     @gmarull
-# General top-level docs
-/doc/nrf/config_and_build/                @greg-fer
-/doc/nrf/installation/                    @greg-fer
-/doc/nrf/security/                        @greg-fer
-/doc/nrf/test_and_optimize/               @greg-fer
-/doc/nrf/*.rst                            @greg-fer
```

Comment:
These I'm watching because of my involvement with Weyland and the ongoing restructuring. Please keep me explicitly on them. Thanks.

Reply:
But you are there, because of the following line:
`/doc/**/*.rst @nrfconnect/ncs-doc-owners`

Reply:
Is there a way I can filter notifications and additions just for these four categories above? I work with 5 different teams in the NCS, and I don't want to be added to doc PRs of the other 34.

Reply:
I am really not sure about that. You probably need to use an email filter or the built-in GitHub web notifications mechanism for that.

Reply:
GitHub web notification cannot really filter. The only routing it can do is to direct messages to additional email addresses based on organization. Filtering in emails is probably not going to work, because the PR titles are rather hard target to get reliable filter on.